src/libraries/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.Directory.Move.cs

```diff
@@ -93,6 +93,27 @@ public void Directory_Move_With_Set_NotifyFilter()
             DirectoryMove_WithNotifyFilter(WatcherChangeTypes.Renamed);
         }
 
+        [Fact]
+        public void FileSystemWatcher_Directory_Move_SynchronizingObject()
+        {
+            TestISynchronizeInvoke invoker = new TestISynchronizeInvoke();
```

Comment:
nit: for all of these tests, you can move this into the using block, just above where you store it into SynchronizingObject.

Reply:
I've moved this for all the tests. I think I probably put it where I did and didn't use `var` because that's how it's done in `tests/FileSystemWatcher.cs`